mappings/net/minecraft/network/packet/s2c/play/UnlockRecipesS2CPacket.mapping

```diff
@@ -2,6 +2,7 @@ CLASS net/minecraft/class_2713 net/minecraft/network/packet/s2c/play/UnlockRecip
 	FIELD field_12408 action Lnet/minecraft/class_2713$class_2714;
 	FIELD field_12409 recipeIdsToInit Ljava/util/List;
 	FIELD field_12414 recipeIdsToChange Ljava/util/List;
+	FIELD field_25797 furnaceFilteringCraftable Lnet/minecraft/class_5411;
```

Comment:
Doubt it, filtering crafterable has been generalized.

Reply:
Definitely, this should be `getRecipeBookOptions` and `recipeBookOptions` now, but I was mainly focused on making the fields consistent with the getter methods.

Reply:
Should be fixed. We shouldn't introduce wrong names.

Reply:
Fair enough